analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NonDerivedPrivateClassesShouldBeSealed.cs

```diff
@@ -50,16 +50,34 @@ private class PrivateDerivedClassSecondExtension : PrivateDerivedClass { }
 
         private class TheThirdExtension : PrivateDerivedClassSecondExtension { } // Noncompliant
 
-        private class PrivateClass // Noncompliant
```

Comment:
was this previously an FP?

Reply:
This particular line no, because this `PrivateClass` was not inherited in any case.